lib/web/apiserver.go

```diff
@@ -803,8 +803,10 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/sites/:site/auth/export", h.authExportPublic)
 	h.GET("/webapi/auth/export", h.authExportPublic)
 
-	// token generation
+	// join token handlers
 	h.POST("/webapi/token", h.WithAuth(h.createTokenHandle))
+	h.GET("/webapi/tokens", h.WithAuth(h.getTokens))
+	h.DELETE("/webapi/tokens/:id", h.WithAuth(h.deleteToken))
```

Comment:
Can we hide the :id in thee DELETE request body instead of exposing it in URL ? 

Let say that a user has right to list tokens but it don't have permission to delete tokens. And a customer infrastructure tools like: 


LB -> Ingress  -> Teleport Proxy 

Please note that `:id == token_secret` in case of static tokens.

When a client calls `/webapi/tokens/token_secret` with missing permission to delete secret the Ingress can log the request HTTP Path and expose token_secret in Ingress logs.

Reply:
Great idea, i'll make the change!

Reply:
Decided to send via a custom header as some servers/load balancers might just ignore any request body sent with a DELETE